module/VuFind/src/VuFind/Db/Service/AccessTokenService.php

```diff
@@ -108,7 +118,14 @@ public function storeNonce(int $userId, ?string $nonce): void
      */
     public function getNonce(int $userId): ?string
     {
-        return $this->accessTokenTable->getNonce($userId);
+        $dql = 'SELECT data '
+            . 'FROM ' . $this->getEntityClass(AccessToken::class) . ' at '
+            . 'WHERE at.user_id = :userId';
```

Comment:
Doctrine uses property names rather than database table rows, so there is no such thing as `user_id`. I believe this should be changed to `user` (assuming that the entity class is also adjusted to use user instead of userId, which I think would be the correct thing to do).

Also note that if you want to turn an integer into a user object for the purposes of making a query, there's a helper function for that: `$user = $this->getDoctrineReference(User::class, $userId);`. (That example assumes that you've included a use statement for `VuFind\Db\Entity\User`).